bank-sdk/example-app/src/main/java/net/gini/android/bank/sdk/exampleapp/ui/ConfigurationActivity.kt

```diff
@@ -616,4 +651,9 @@ class ConfigurationActivity : AppCompatActivity() {
         }
     }
 
+    private fun checkClientSecretAndClientId() {
```

Comment:
[Suggestion] I think if we rename the method to `applyClientSecretAndClientId()` it reflects more accurately what it does.